src/backend/server.py

```diff
@@ -0,0 +1,248 @@
+
+    id_user = db.Column(db.Integer, primary_key=True)
+    firstName = db.Column(db.Text, nullable=False)
+    lastName = db.Column(db.Text, nullable=False)
+    dob = db.Column(db.Text, nullable=False)
+    email = db.Column(db.Text, nullable=False, unique=True)
+    password = db.Column(db.Text, nullable=False)
+    gender = db.Column(db.Text, nullable=False)
+    age = db.Column(db.Text, nullable=False)
+    properties = db.Column(db.Text)
+    autherized_to = db.Column(db.Integer)
+
+    def __init__(self, firstName, lastName, dob, email, password, gender, age, properties, autherized_to):
+        self.firstName = firstName
+        self.lastName = lastName
+        self.dob = dob
+        self.email = email
+        self.password = password
+        self.gender = gender
+        self.age = age
+        self.properties = properties
+        self.autherized_to = autherized_to
+
+    def get_id(self):
+        return self.id_user
+
+
+class properties(db.Model):
+
+    id_property = db.Column(db.Integer, primary_key=True)
+    street = db.Column(db.Text, nullable=False)
+    city = db.Column(db.Text, nullable=False)
+    state = db.Column(db.Text, nullable=False)
+    zipcode = db.Column(db.Integer, nullable=False)
+    description = db.Column(db.Text, nullable=False)
+    estimate = db.Column(db.Integer, nullable=False)
+    photos = db.Column(db.Text, nullable=False)
+    videos = db.Column(db.Text, nullable=False)
+    belongs_to = db.Column(db.Integer, db.ForeignKey(
+        'users.id_user'), nullable=False)
+
+    users = db.relationship('users', backref='user', uselist=False)
+
+    def __init__(self, street, city, state, zipcode, description, estimate, photos, videos, belongs_to):
+        self.street = street
+        self.city = city
+        self.state = state
+        self.zipcode = zipcode
+        self.description = description
+        self.estimate = estimate
+        self.photos = photos
+        self.videos = videos
+        self.belongs_to = belongs_to
+
+
+class photos(db.Model):
+
+    id_photo = db.Column(db.Integer, primary_key=True)
+    path = db.Column(db.Text, nullable=False)
+    belongs_to = db.Column(db.Integer, db.ForeignKey(
+        'properties.id_property'), nullable=False)
+
+    properties = db.relationship(
+        'properties', backref='photo_property', uselist=False)
+
+    def __init__(self, path, belongs_to):
+        self.path = path
+        self.belongs_to = belongs_to
+
+
+class videos(db.Model):
+
+    id_video = db.Column(db.Integer, primary_key=True)
+    path = db.Column(db.Text, nullable=False)
+    belongs_to = db.Column(db.Integer, db.ForeignKey(
+        'properties.id_property'), nullable=False)
+
+    properties = db.relationship(
+        'properties', backref='video_property', uselist=False)
+
+    def __init__(self, path, belongs_to):
+        self.path = path
+        self.belongs_to = belongs_to
+
+
+@app.route("/hello")
+def hello():
+    return jsonify({'string': 'Hello World'})

```

Comment:
@jonbondamon  This jsonify returns an entire object.... its a submodule of the flask framework. Figured this out after trying to send back a normal string and it not working... Based on this, we should be able to return objects and reference them as need be... Gonna be testing this a bit more